importexport/musicxml/importmxmlpass2.cpp

```diff
@@ -4554,8 +4559,10 @@ Note* MusicXMLParserPass2::note(const QString& partId,
                                     const auto track = msTrack + msVoice;
                                     const auto extraRest = addRest(_score, measure, noteStartTime + dura, track, msMove,
                                                                    TDuration { missingCurr* tuplet->ratio() }, missingCurr);
-                                    extraRest->setTuplet(tuplet);
-                                    tuplet->add(extraRest);
+                                    if (extraRest) {
+                                          extraRest->setTuplet(tuplet);
+                                          tuplet->add(extraRest);
+                                          }
```

Comment:
These changes are in 3.5rc and in 3.x, so not an issue, but why then are they listed here?